lib/view_component/form/builder.rb

```diff
@@ -55,8 +55,14 @@ def submit(value = nil, options = {})
         render_component(:submit, self, @object_name, value, options)
       end
 
-      # def button(value = nil, options = {}, &block)
-      # end
+      def button(value = nil, options = {}, &block)
+        if value.is_a?(Hash)
+          options = value
+          value = nil
+        end
+        value ||= submit_default_value
+        render_component(:button, self, @object_name, value, options, &block)
+      end
```

Comment:
Maybe we could remove `@object_name` like for submit

Reply:
Done in 587b550